### PR TITLE
updates to support v0.2

### LINKED ISSUE #6: flip resource and project in slug

### PR DESCRIPTION
**Pull request by neutrinog**
Fixes #6, #5, #4, #3.

@jag3773  Accidentally changed images to a full type as well. Should we keep that?

Fixed a bunch of typos in the documentation as well.

### CODE REVIEW

**Comment by neutrinog** [comment]
@jag3773 I have some other questions:

* I'm wondering if we should move the checks_performed out of the resource and place it directly in the resource container. Because it can vary be resource container.
* also in switching the order of projects and resources is seems that what is currently the resource `status` would be better suited in the project or perhaps we need both and the one in the resource is the lowest common denominator, or separate altogether. It seems to me you might be translating projects using different source text which may have different licenses.

File: docs/manifest.rst [hunk 1]
```diff
@@ -53,62 +53,55 @@ All resource containers have a package.json file which looks something like this
           "source_translations": [
             {
               "language_slug": "en",
-              "resource_slug": "asv",
-              "version": "1901"
+              "resource_slug": "obs",
+              "version": "3"
```
Comment (jag3773) [inline docs/manifest.rst]: You can't do this either...

File: docs/manifest.rst [hunk 2]
```diff
@@ -20,7 +21,7 @@ All resource containers have a package.json file which looks something like this
       "project": {
         "slug": "gen",
         "name": "Genesis",
-        "description": "",
+        "desc": "",
```
Comment (jag3773) [inline docs/manifest.rst]: No reason to make this change, right?

File: docs/manifest.rst (hunk 1 above)
Comment (neutrinog) [inline docs/manifest.rst]: oops!

File: docs/manifest.rst [hunk 3]
```diff
@@ -44,7 +44,7 @@ All resource containers have a package.json file which looks something like this
             "Wycliffe Associates"
           ],
           "pub_date": "2015-12-17",
-          "license": "CC BY-SA 4.0",
+          "license": "CC BY-SA",
```
Comment (jag3773) [inline docs/manifest.rst]: Yes.

**Pull request merged**